compile_clang.sh

```diff
@@ -12,7 +15,7 @@ echo "==> Building $_defconfig"
 make $_make_vars $_defconfig
 
 echo "==> Building $_targets with clang"
-make $_make_vars CROSS_COMPILE="$_cross_compile" CC="${_clang_path}/clang" CLANG_TRIPLE=aarch64-linux-gnu $_targets
+make $_make_vars CC=clang CLANG_TRIPLE=aarch64-linux-gnu- CROSS_COMPILE=aarch64-linux-android- CROSS_COMPILE_ARM32=arm-linux-androideabi- $_targets
```

Comment:
I don't think `CLANG_TRIPLE=aarch64-linux-gnu-` was supposed to get that extra trailing hyphen?